troubleshooting/troubleshooting.md

```diff
@@ -121,14 +121,15 @@ If you are using Linux, you must install a few dependencies for the video to wor
 
     apt install gstreamer1.0-plugins-bad gstreamer1.0-libav
 
-If your camera is detected, it will be listed under the USB devices on the Companion system webpage. If your camera is not detected, make sure that the camera supports H.264 video output, and make sure the usb cable is well-seated, or try another USB cable.
+Verify that your camera is detected by seeing it listed under the detected video devices section on the Companion [system webpage](/operators-manual/companion-web.md#system). If your camera is not detected, make sure that the camera supports H.264 video output, and make sure the cable is well-seated, or try another camera cable.
```

Comment:
Should we show that a camera is connected but does not support H264 in companion ?
E.g:
- /dev/video0 (Camera does not provide H264 output, will not be used)

This will help the user to identify if the camera is connected or does not provide H264.

Reply:
good idea
https://github.com/bluerobotics/companion/issues/229